source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimens.sql

```diff
@@ -0,0 +1,26 @@
+﻿CREATE TABLE [dbo].[UnmatchedSpecimens]
+(
+	[ReferenceLaboratoryNumber] NVARCHAR(50) NOT NULL PRIMARY KEY,
+	[SpecimenDate] DATETIME,
+	[SampleType] NVARCHAR(255),
+	[LaboratoryName] NVARCHAR(65),
+	[ReferenceLaboratory] NVARCHAR(65),
+    [Species] VARCHAR(30),
```

Comment:
It looks like something weird may be happening with the whitespace in this file - a mix of tabs and spaces perhaps?